app/assets/javascripts/views/listView.js

```diff
@@ -0,0 +1,22 @@
+var ListView = function(list){
+  this.list = list;
+
+  this.$el = $("<div class='list'></div>");
+  this.render();
+
+  $("main").append(this.$el);
```

Comment:
I think this may be the reason your view isn't rendering, have you tried putting this line of code into the render function itself? Though i imaging `.render()` is synchronous